### DISCUSSION TITLE
Thank you for the motivation

### DISCUSSION THREAD

**Discussion by kai-tub** [pradyunsg/furo#373] [Show and tell]
I just stumbled upon this little gem:
https://github.com/pradyunsg/furo/blob/3bcbe2ce404080061481638efff8e6102b3b812d/src/furo/theme/furo/sidebar/brand.html#L5-L11

I just wanted to say thank you; I laughed quite a bit. :D 
I am trying to create my own sphinx theme and am looking up the source code of different themes that I really like.
Even if I am having a hard time with Sphinx...
This definitely helped me to keep my spirit up :+1: 

PS: Also thank you for stb ;)

**Comment by kai-tub** [comment]
And it keeps getting better :D :heart: 

https://github.com/pradyunsg/furo/blob/3bcbe2ce404080061481638efff8e6102b3b812d/src/furo/navigation.py#L24

**Comment by pradyunsg** [comment]
Thank you for reaching out and sharing the positivity! Much appreciated! ^>^

> PS: Also thank you for stb ;)

You're welcome! :)
